src/pages/assign-calls.js

```diff
@@ -55,7 +60,12 @@ export default function AssignCallsPage() {
     };
 
     const handleAssign = async (event) => {
-        if (selectedAssignment.length == 0 || selectedCallHandlers.length == 0) {
+        if (
+            selectedAssignment.length == 0 ||
+            selectedCallHandlers.length == 0 ||
+            selectedCallType == '' ||
+            selectedCallType == 'Please choose'
```

Comment:
We could extract the conditions in this if statement so it becomes:

 if formIsValid() .. 
 
 A minor improvement for readability / separation of functions.

Reply:
I agree, I'll add this in :)